.svglintrc.js

```diff
@@ -569,6 +569,31 @@ module.exports = {
                 reporter.error(`${errorMsg}: ${reason}`);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = 'svg-format';
+
+            // Don't allow explicit '</path>' closing tag
+            const explicitClosingPathIndex = ast.source.indexOf('</path>');
+            if (explicitClosingPathIndex > -1) {
+              const reason = `found explicit closing "path" tag at index ${explicitClosingPathIndex}.`
+                          + ' The syntax \'></path>\' should be replaced by \'/>\'.';
+              reporter.error(`Invalid SVG content format: ${reason}`);
+            }
```

Comment:
I've never heard of the term "explicit closing tag" and couldn't really find it being used in a _credible_ way in my quick 2 minutes Google search. Maybe we can instead use the term "self-closing" tag, e.g. rewrite it along the lines of

```suggestion
            // Don't allow explicit '</path>' closing tag
            const pathClosingTagIndex = ast.source.indexOf('</path>');
            if (pathClosingTagIndex > -1) {
              const reason = `found a closing "path" tag at index ${explicitClosingPathIndex}.`
                          + ' The path should be self-closing, use  \'/>\' instead of \'></path>\'.';
              reporter.error(`Invalid SVG content format: ${reason}`);
            }
```

Also wondering if we can use [`String.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) here, but I'm not sure if `ast.source` is a string.

Reply:
> Also wondering if we can use [`String.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) here, but I'm not sure if `ast.source` is a string.

Done in https://github.com/simple-icons/simple-icons/pull/6513/commits/bb7847863e624e0f6e208fad555a915b5c9fef57